src/utils/session.ts

```diff
@@ -202,7 +202,7 @@ export async function unsealSession(
   return unsealed;
 }
 
-export async function clearSession(event: H3Event, config: SessionConfig) {
+export async function clearSession(event: H3Event, config: Partial<SessionConfig> = {}) {
```

Comment:
Making _name_ also optional, can be both good DX but also lead to programmer mistakes forgetting to provide custom name. (also might be needed to clear cookie from subpath)

Reply:
Maybe only add Partial to remove password requirement?